google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunnerImpl.java

```diff
@@ -625,6 +635,24 @@ public void onTransactionMetadata(Transaction transaction, boolean shouldInclude
       }
     }
 
+    /**
+     * In read-write transactions, the precommit token with the highest sequence number from this
+     * transaction attempt will be tracked and included in the
+     * [Commit][google.spanner.v1.Spanner.Commit] request for the transaction.
+     */
+    @Override
+    public void onPrecommitToken(MultiplexedSessionPrecommitToken token) {
+      if (token == null) return;
```

Comment:
nit: always use curly braces for `if` statements. See https://google.github.io/styleguide/javaguide.html#s4.1-braces

Reply:
Done